compiler/resolution/wrappers.cpp

```diff
@@ -216,8 +219,11 @@ buildDefaultWrapper(FnSymbol* fn,
       ArgSymbol* wrapper_formal = copyFormalForWrapper(formal);
       if (fn->_this == formal)
         wrapper->_this = wrapper_formal;
-      if (formal->hasFlag(FLAG_IS_MEME))
-        wrapper->_this->defPoint->insertAfter(new CallExpr(PRIM_MOVE, wrapper->_this, wrapper_formal)); // unexecuted none/gasnet on 4/25/08
+      if (formal->hasFlag(FLAG_IS_MEME)) {
+        if (wrapper->_this != NULL) {
```

Comment:
_this is used in every method.  Constructors are the only method that don't use it as an argument, they instead insert it as a variable and copy from "meme" into it - they do that so that checkNormalize can ensure we don't reference a field in the constructor argument list.

In short, I don't think we can get away from _this quite yet